src/core/withExperiment.js

```diff
@@ -64,13 +67,19 @@ export const withExperiment = ({
         return;
       }
 
-      const { cookies, randomizer, variantA, variantB } = this.props;
+      const { cookies, id, randomizer, variantA, variantB } = this.props;
 
       this.experimentCookie = cookies.get(this.getCookieName());
 
       if (this.experimentCookie === undefined) {
-        this.experimentCookie = randomizer() >= 0.5 ? variantA : variantB;
+        const variant = randomizer() >= 0.5 ? variantA : variantB;
+        this.experimentCookie = variant;
         cookies.set(this.getCookieName(), this.experimentCookie, cookieConfig);
+        // send an enrollment event
+        _tracking.sendEvent({
+          action: variant,
+          category: `AMO Experiment enrollment - id: ${id}`,
```

Comment:
I suppose we could use a constant for this string and only append `id`:

```
const EXPERIMENT_ENROLLMENT_CATEGORY = 'AMO Experiment Enrollment –';

[...]

category: [EXPERIMENT_ENROLLMENT_CATEGORY, id].join(' '),
```

Reply:
Yeah, then we could re-use that in the test. Thanks, I'll make that change.